Added hide_npcs wiki description for clarification

majster2nn opened the pull request:
<!-- Please describe your changes here. -->
I added a simple description for `hide_npcs` section in Integration docs since there was none and some people may find it confusing and difficult. I know that there is a ton of discord threats about it but i think that a simple (not perfect) description will make it more clear for people to understand this mechanic.

---

### Related Issues

<!-- Issue number if existing. -->
Closes #XXXX

### Requirements
- [x] I made sure my contribution fulfills the [requirements](https://betonquest.org/DEV/Participate/Process/Submitting-Changes/#checklist).

### Reviewer's checklist

<!-- DON'T DO ANYTHING HERE -->
<!-- This is a checklist for the reviewers, and will be checked by them! -->
Did the contributor...
- [x]  ... test their changes?
- [x]  ... increment the [Version](https://betonquest.org/DEV/Participate/Misc/Versioning-and-Releasing/#versioning)?
- [x]  ... update the [Changelog](https://betonquest.org/DEV/Participate/Process/Maintaining-the-Changelog/)?
- [x]  ... update the [Documentation](https://betonquest.org/DEV/Participate/Process/Docs/Workflow/)?
- [x]  ... adjust the [ConfigPatcher](https://betonquest.org/DEV/API/ConfigurationFiles/#updating-configurationfiles)?
- [x]  ... clean the commit history?

Check if the build pipeline succeeded for this PR!

TheosRee commented:
Thanks for your contribution!
First, the other two Npc integrations don't get an improvement, but that is kinda redundant because they link to the `Npc` page.
And here is the point: Instead of adding a full example in the Citizens Integration part I think it would be better if you would link the actual documentation for that feature (for example before the info box) and add the full example there.
Also, I changed how you address the Npcs to the ID system so you would need to adjust your changes to that. You can also use the citizens npcs in the example like we do in the tutorial.

majster2nn commented:
I changed it a bit and now the citizens integration points to the npc hiding wiki page and i added full examples there with a little change in description that was already there.
